python_modules/dagster/dagster/_core/pipes/context.py

```diff
@@ -56,6 +67,14 @@
 PipesExecutionResult: TypeAlias = Union[MaterializeResult, AssetCheckResult]
 
 
+class PipesLaunchedData(TypedDict):
+    """Payload generated on the Client side after the external process startup
```

Comment:
"Payload generated in the orchestration process after external process startup" (not "Client side")

Reply:
:melting_face: